torch/serialization.py

```diff
@@ -1085,11 +1085,11 @@ def _check_container_source(container_type, source_file, original_source):
                         if file_size == 0:
                             f.write(lines)
                         elif file_size != len(lines) or f.read() != lines:
-                            raise IOError
+                            raise OSError
```

Comment:
Can you explain the rationale behind replacing IOError with OSError?

Reply:
It's an autofix. According to https://beta.ruff.rs/docs/rules/os-error-alias/, IOError is an alias of OSError